### PR TITLE
Add info to remove encryption for outgoing mail

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross  to review this

### CODE REVIEW

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 1]
```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
 
 8. Choose **Save**.
 
+## Create mail flow rules to remove encryption for outgoing email messages with the new OME capabilities
```
Comment (KCCross) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md:92]: Create a mail flow rule to remove encryption for outgoing email message by using the new OME capabilities. Also under what conditions would they want to remove encryption from outgoing mail? This seems... unlikely.

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 2]
```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
...
+
+You can define mail flow rules for triggering remove message encryption with the new OME capabilities by using the EAC.
```
Comment (KCCross) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md:94]: You can create a mail flow rule to remove message encryption with the new OME capabilities by using the EAC.

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 3]
```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
...
+
+### Use the EAC to create a rule for removing encryption from email messages with the new OME capabilities
```
Comment (KCCross) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]: Use the EAC to create a rule to remove encryption from email messages with the new OME capabilities

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 4]
```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
...
+
+1. In a web browser, using a work or school account that has been granted admin permissions, [sign in to Office 365](https://support.office.com/article/b9582171-fd1f-4284-9846-bdd72bb28426#ID0EAABAAA=Web_browser).
```
Comment (KCCross) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]: "that has been granted global administrator permissions,"

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 5]
```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
...
+
+2. Choose the **Admin** tile.
+
+3. In the Microsoft 365 admin center, choose **Admin centers** \> **Exchange**.
+
+4. In the EAC, go to **Mail flow** \> **Rules** and select **New** ![New icon](../media/457cd93f-22c2-4571-9f83-1b129bcfb58e.gif) \> **Create a new rule**. For more information about using the EAC, see [Exchange admin center in Exchange Online](https://docs.microsoft.com/exchange/exchange-admin-center).
+
+5. In **Name**, type a name for the rule, such as Remove encryption from incoming mail.
```
Comment (KCCross) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]: Shouldn't this be "Remove encryption from outgoing mail."? That's what the rest of this new procedure says.

**Comment by KCCross** [comment]
Samson, please review changes and make necessary updates. Thanks!!

**Comment by samschan-ms** [comment]
@KCCross  added all you changes

**Comment by samschan-ms** [comment]
@KCCross  please close this for me.

**Comment by KCCross** [comment]
All good. ty Samson.

**Pull request merged**